Source/Core/DotSpatial.Data.Forms/DotSpatial.Data.Forms.csproj

```diff
@@ -12,4 +13,31 @@
     <ProjectReference Include="..\DotSpatial.Symbology\DotSpatial.Symbology.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Update="DataFormsMessageStrings.Designer.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>DataFormsMessageStrings.resx</DependentUpon>
+    </Compile>
+    <Compile Update="DialogImages.Designer.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>DialogImages.resx</DependentUpon>
+    </Compile>
+    <Compile Update="DirectoryView.cs">
+      <Generator>PublicResXFileCodeGenerator</Generator>
+    </Compile>
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Update="DataFormsMessageStrings.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>DataFormsMessageStrings.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
+    <EmbeddedResource Update="DialogImages.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>DialogImages.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
```

Comment:
Why did you add those lines?
VS 2022 shows this information without having to add it to the csproj file.